src/api/spec/components/patchinfo_component_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe PatchinfoComponent, :vcr, type: :component do
```

Comment:
```suggestion
RSpec.describe PatchinfoComponent, type: :component do
```
These specs seem not to need VCR.

Reply:
True! Removed!